### DISCUSSION TITLE
Container seems running, but the server is not accessible [Fabric]

### DISCUSSION THREAD

**Discussion by Sallos725** [itzg/docker-minecraft-server#2443] [Q&A]
Hi. I was able to run several curseforge servers thanks to this awesome docker image. 
However, it became a huge challenge to open the fabric server.
Currently I am using portainer to manage my docker compose files and servers, or direct ssh to the server for the generic packs.
Long story short, the compose file seems to run the container, but I am not able to access through the minecraft client via this compose file:

<details>
<summary> docker-compose.yml </summary>
<pre>
version: "3.8"
<br>
services:
    mincraft:
    image: itzg/minecraft-server:java17
    container_name: ca
    ports:
      - "30000:30000"
    volumes:
      - "/home/{my name}/ca:/data"
    stdin_open: true
    tty: true
    environment:
      # Generals
      TZ: "Seoul/Asia"
      EULA: "TRUE"
      # JVM
      INIT_MEMORY: "6G"
      MAX_MEMORY: "8G"
      USE_AIKAR_FLAGS: "TRUE"
      GUI: "FALSE"
      # In-Game Thinggy
      MODE: "survival"
      DIFFICULTY: "normal"
      MAX_PLAYERS: "6"
      NETWORK_COMPRESSION_THRESHOLD: "256"
      SIMULATION_DISTANCE: "8"
      VIEW_DISTANCE: "16"
      SPAWN_PROTECTION: "0"
      ALLOW_FLIGHT: "TRUE"
      # RCON
      ENABLE_RCON: "false"
      RCON_PASSWORD: "{I have changed password just in case}"
      # Misc
      SERVER_NAME: "Create Astral: M1NDB3ND3R strikes back"
      MOTD: "Welcome! to M1ND-B3ND3D server"
      OPS: "_me!_"
      SNOOPER_ENABLED: "FALSE"
      # Mods
      TYPE: "AUTO_CURSEFORGE"
      CF_API_KEY: "{API key goes here, with dollar sign before the pre-generated dollar sign}"
      CF_PAGE_URL: "https://www.curseforge.com/minecraft/modpacks/create-astral"
      # Autopause
      ENABLE_AUTOPAUSE: "TRUE"
      MAX_TICK_TIME: "-1"
      AUTOPAUSE_TIMEOUT_INIT: "300"
      AUTUPAUSE_TIMEOUT_EST: "30"
    restart: unless-stopped
</pre>
</details>

The server pack told me to put two additional mods from the client, so I have put them in the correct mod directory which is ~/ca/mods/ .

Is there anything I have missed?

**Comment by itzg** [comment]
The container side of the port mapping should remain as 25565, such as

```
ports:
      - "30000:25565"
```

**Reply by Sallos725** [reply]
Since I am running another server via port 25565, I tried to open the server through 30000. 
I also managed to connect to port 30000 when I was trying to connect to the server. 
Since then, what environment should I write? 25565:30000? or 30000:25565?
